grammars/verilog.treetop

```diff
@@ -828,15 +830,17 @@ module OrigenVerilog
 
       # dimension ::= [ dimension_constant_expression : dimension_constant_expression ]
       rule dimension
-        "[" s dimension_constant_expression s ":" s dimension_constant_expression s "]" {
+        ("[" s dimension_constant_expression s ":" s dimension_constant_expression s "]" s "[" s dimension_constant_expression s ":" s dimension_constant_expression s "]" /
+         "[" s dimension_constant_expression s ":" s dimension_constant_expression s "]") {
```

Comment:
This is converting this rule from defining a single dimension to defining multiple dimensions.

Is the proper update not to change `dimension` somewhere to `dimension+` so that it will take multiple?